### PR TITLE
Add review details page

### PR DESCRIPTION
**Pull request by adiabramovitch**
updated
image

### CODE REVIEW

File: src/Components/Wizard/steps/ReviewDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          Review the information below and click Finish to complete
+          provisioning.
+        </Title>
+      </FormGroup>
+      <FormGroup>
+        <ExpandableAWS />
+      </FormGroup>
+      <FormGroup></FormGroup>
+      <FormGroup></FormGroup>
+      <FormGroup></FormGroup>
```
Comment (amirfefer) [inline src/Components/Wizard/steps/ReviewDetails/index.js:24]: leftover?

File: src/Components/Wizard/steps/ReviewDetails/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Form, FormGroup, Title } from '@patternfly/react-core';
+import { ExpandableAWS } from '../../../ExpandableAWS';
+
+const ReviewDetails = () => {
+  return (
+    <Form isWidthLimited maxWidth="80%">
```
Comment (amirfefer) [inline src/Components/Wizard/steps/ReviewDetails/index.js:7]: do we need to wrap the review step with a `<Form>`? basically, it's a read-only review with no inputs. no?

**Review by amirfefer — COMMENTED** [review]
Thanks @adiabramovitch! Looks good, just some minor inline comments

File: src/Components/Wizard/steps/ReviewDetails/index.js (hunk 2 above)
Comment (ezr-ondrej) [reply src/Components/Wizard/steps/ReviewDetails/index.js:7]: I tend to agree here :thinking:

Comment (adiabramovitch) [reply src/Components/Wizard/steps/ReviewDetails/index.js:7]: yes, I agree, so what is it supposed to be? 
Text Input? 
I need to warp the ExpandableAWS and the two titles somehow

Comment (ezr-ondrej) [reply src/Components/Wizard/steps/ReviewDetails/index.js]: Why do you need to wrap? wouldn't empty component `<></>` be enough there?

Comment (adiabramovitch) [reply src/Components/Wizard/steps/ReviewDetails/index.js]: I changed it to Stack and each component inside is inside a Stack item. 
I also updated the screenshot

**Review by ezr-ondrej — APPROVED** [review]
Thanks @adiabramovitch ! :+1:

**Pull request merged**